docs/markdown/Machine-files.md

```diff
@@ -8,10 +8,53 @@ environments](Native-environments.md).
 ## Sections
 
 The following sections are allowed:
+- constants
 - binaries
 - paths
 - properties
 
+### constants
+
+*Since 0.55.0*
+
+String and list concatenation is supported using the `+` operator.
+Entries defined in the `[constants]` section can be used in any other section,
+entries in any other section can be used only within that same section and only
+after it has been defined.
```

Comment:
Probably should document that `[constants]` is guaranteed to be read before any other sections.

Reply:
done.